wikibaseintegrator/entities/baseentity.py

```diff
@@ -239,17 +239,17 @@ def _write(self, data: Dict = None, summary: str = None, login: _Login = None, a
 
         try:
             json_result: dict = mediawiki_api_call_helper(data=payload, login=login, allow_anonymous=allow_anonymous, is_bot=is_bot, **kwargs)
-
+        except Exception:
+            print('Error while writing to the Wikibase instance')
```

Comment:
Could we have this message go to log.warning instead to avoid confusing the user? (I'm deliberately trying to upload items without waiting for SPARQL and use the returned QID if I get the 'wikibase-validator-label-with-description-conflict' error.